CHANGELOG.md

```diff
@@ -6,8 +6,14 @@
   [#210](https://github.com/oddbird/true/issues/210)
 - INTERNAL: Limit files included in npm package
   [#189](https://github.com/oddbird/true/issues/189)
+- INTERNAL: Convert JS to TypeScript and bundle type definitions
+  [#212](https://github.com/oddbird/true/issues/212) --
+  thanks to [@robertmaier](https://github.com/robertmaier) for the initial PR
+  [#206](https://github.com/oddbird/true/pull/206)
+- INTERNAL: Remove documentation static-site from True repository
+- INTERNAL: Drop internal testing with Mocha
```

Comment:
Curious about this one. Are we using a different test runner? (This should maybe get some clarification here?)